cmd2/cmd2.py

```diff
@@ -466,11 +462,20 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, persistent
         # If this string is non-empty, then this warning message will print if a broken pipe error occurs while printing
         self.broken_pipe_warning = ''
 
+        # Commands that will run at the beginning of the command loop
+        self._startup_commands = []
```

Comment:
Changing to the use of the `self._startup_commands` list instead of using `self.cmdqueue` inherited from `cmd` solves the "Command queue issue" alluded to in the comments.

Overall, I think we should do a better job in the Sphinx docs of calling out the elements in `cmd` which are not used or supported in `cmd2`.  We have a "Note" for this in our [Overview](https://cmd2.readthedocs.io/en/latest/overview.html).  But I think it is long overdue that we expand upon that.